### ISSUE TITLE
Have Linux download give Flatpak option

### ISSUE THREAD

**Issue by MMachado05** [Ryujinx/Ryujinx-Website#33]
The Flatpak is seemingly officially supported on FlatHub, yet when the install button for Linux is selected, it downloads a tarball. Newer Linux users may have a hard time understanding what to do with it, and the Flatpak format is objectively far easier to maintain for end users. I think the Flatpak option should be prioritized (perhaps saying "download on Linux from Flathub") and the tarball can still be offerred for those who want to use that version for personal reasons.

**Comment by marysaka** [comment]
> The Flatpak is seemingly officially supported on FlatHub, yet when the install button for Linux is selected, it downloads a tarball. Newer Linux users may have a hard time understanding what to do with it, and the Flatpak format is objectively far easier to maintain for end users. I think the Flatpak option should be prioritized (perhaps saying "download on Linux from Flathub") and the tarball can still be offerred for those who want to use that version for personal reasons.

I agree too! If you want to do a PR for that feel free to add it, I could probably look into that once I find some time again

**Comment by MMachado05** [comment]
I would love to, but unfortunately I'm terribly unfamiliar with TypeScript 😅. Glad to see it's viewed as a good enhancement, though!
